src/eko/thresholds.py

```diff
@@ -164,21 +170,29 @@ def from_dict(cls, theory_card, prefix="k", max_nf_name="MaxNfPdf"):
                 run card with the keys given at the head of the :mod:`module <eko.thresholds>`
             prefix : str
                 prefix for the ratio parameters
+            masses: list
+                list of :math:`\overline{MS}` masses squared or None if POLE masses are used
```

Comment:
This MSbar has a different style than that used somewhere else:
```
:math:`\bar{MS}`
```
We definitely need an abbreviation, even to make it uniform.